apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx

```diff
@@ -0,0 +1,860 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import EnvelopeIcon from "@heroicons/react/24/outline/EnvelopeIcon";
+import EnvelopeSolid from "@heroicons/react/24/solid/EnvelopeIcon";
+import ArrowDownTrayIcon from "@heroicons/react/24/outline/ArrowDownTrayIcon";
+
+import { getLocale } from "@gc-digital-talent/i18n";
+import { Heading, Link, Well } from "@gc-digital-talent/ui";
+
+import logoImg from "~/assets/img/iap-logo.svg";
+import heroImg from "~/assets/img/IAPManager-Hero.webp";
+import section1Img from "~/assets/img/IAPManager-Section-01.webp";
+import section1FlourishImg from "~/assets/img/IAPManager-Section-01-flourish.webp";
+import section2Img from "~/assets/img/IAPManager-Section-02.webp";
+import section2FlourishImg from "~/assets/img/IAPManager-Section-02-flourish.webp";
+import testimonials from "~/assets/img/IAPManager-Testimonials.webp";
+import section5Img from "~/assets/img/IAPManager-Section-05.webp";
+import section5FlourishImg from "~/assets/img/IAPManager-Section-05-flourish.webp";
+import packageEnDoc from "~/assets/documents/ITAPIP.Manager.Package.Text_FINAL_with.frame.docx";
+
+import { CloseQuote, OpenQuote, Triangle } from "../IAPHomePage/components/Svg";
+import TopRightFrame from "./components/Svg/TopRightFrame";
+
+const placeholderUrl = "about:blank";
+const makeLink = (chunks: React.ReactNode, url: string) => (
+  <Link href={url} color="primary">
+    {chunks}
+  </Link>
+);
+
+function buildMailtToUri(
+  emailAddres: string,
```

Comment:
Should be 
buildMailToUri (remove the 't' after 'Mail')
emailAddress (missing 's')
right?